src/java/edu/ucsd/library/dams/jhove/MyJhoveBase.java

```diff
@@ -58,8 +58,8 @@
 	"the President and Fellows of Harvard College. " +
 	"Released under the GNU Lesser General Public License.";
 	
-	private static final String _moduleNames[] = {"PDF-hul","ASCII-hul","GIF-hul","TIFF-hul","WAVE-hul","XML-hul","HTML-hul","ZIP-ptc","GZIP-ptc","BYTESTREAM","BYTESTREAM"};
-	private static final String _fileExten[]   = {".pdf",".txt",".gif",".tif",".wav",".xml",".html",".zip",".gz",".mov",".hierarchy"};
+	private static final String _moduleNames[] = {"PDF-hul","ASCII-hul","GIF-hul","TIFF-hul","WAVE-hul","XML-hul","HTML-hul","ZIP-ptc","GZIP-ptc","BYTESTREAM","BYTESTREAM","BYTESTREAM","BYTESTREAM"};
```

Comment:
Yes, just followed the original pattern without much consideration. I've change the implementation for it, see https://github.com/ucsdlib/damsrepo/pull/44/commits/5f7ed3b8a96a6b922536e2ca9b8a331d31bda6f3 .